crates/bevy_input/src/keyboard.rs

```diff
@@ -88,8 +88,8 @@ use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
 ///
 /// The event is consumed inside of the [`keyboard_input_system`]
 /// to update the [`ButtonInput<KeyCode>`](ButtonInput<KeyCode>) resource.
-#[derive(Event, Debug, Clone, PartialEq, Eq)]
-#[cfg_attr(feature = "bevy_reflect", derive(Reflect), reflect(Debug, PartialEq))]
+#[derive(Event, Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "bevy_reflect", derive(Reflect), reflect(Debug, PartialEq, Hash))]
```

Comment:
```suggestion
#[cfg_attr(
    feature = "bevy_reflect",
    derive(Reflect),
    reflect(Debug, PartialEq, Hash)
)
```

Reply:
Copied from the lint output - untested, but I'm assuming this will likely fix the issue fine @alice-i-cecile 
(I'm waiting on this PR to land for some changes I want to make in bevy_ratatui)